strawberry/schema/config.py

```diff
@@ -1,14 +1,19 @@
 from __future__ import annotations
 
 from dataclasses import InitVar, dataclass, field
+from typing import List
 
-from .name_converter import NameConverter
+from strawberry.description_source import DescriptionSource
+from strawberry.schema.name_converter import NameConverter
 
 
 @dataclass
 class StrawberryConfig:
     auto_camel_case: InitVar[bool] = None
     name_converter: NameConverter = field(default_factory=NameConverter)
+    description_sources: List[DescriptionSource] = field(
```

Comment:
I considered that, but I've chosen a list so that the ordering can be used to set different priorities:

E.g.:
```python
@strawberry.type(description="Strawberry Description")
class Foo:
   """ Docstring description"""
   bar: int

will return different docstrings for `description_sources = [STRAWBERRY_DESCRIPTIONS, TYPE_DOCSTRINGS]` vs `[TYPE_DOCSTRINGS, STRAWBERRY_DESCRIPTIONS]`

Reply:
Hmmm, is that desirable? Not sure if having fallback-style ordering is necessary/desirable if attributes are _also_ inheriting from their scope-ings. What do you think @patrick91 ?

Reply:
Okay, I've changed to enum.Flags and it does look better.

However we now need to be extra conscious about the priorities predefined in code.

In particular, it currenclt assumes: description > resolver > class > attribute
:thinking: Maybe attribute docstrings have priority over class docstrings?